wwudora.module

```diff
@@ -14,3 +14,31 @@ function wwudora_islandora_oai_get_xsl_files() {
     "$module_path/xml/wwu_mods_to_dc.xsl" => 'wwu_mods_to_dc.xsl',
   );
 }
+
+/**
+ * Implements hook_islandora_compound_object_children_added_to_parent().
+ */
+function wwudora_islandora_compound_object_children_added_to_parent($children, $parent_pids) {
+  $parent = islandora_object_load(current($parent_pids));
+  foreach ($children as $child) {
+    module_load_include('inc', 'wwudora', 'includes/derivatives');
```

Comment:
This should just go at the top of the function. Otherwise we're incurring a lot of additional overhead by calling this on every loop iteration.